generator/payload.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  AllUpdates,
+  bpsToRayOrKeepCurrent,
+  boolOrKeepCurrent,
+  valueOrKeepCurrent,
+  Networks,
+  NetworkUpdate,
+} from './index.js';
```

Comment:
This is referencing the compiled output

Reply:
this is unfortunately (as far as I could tell) needed to support ejs "modules", which is needed by some of the dependency packages